mtf/metadata/README.md

```diff
@@ -0,0 +1,157 @@
+    * I want to select some cases based on relevancy
+* __US2:__ Find/__Agregate__ uncovered parts in testsuite to write new tests, or report status of testsuite to managers
+    * I want to see acual coverage for component
+    * I want to see uncovered parts
+* __US3:__ General format allows us more, Describing tests inside source code is `programming lang` specific and harder to handle
+    * I want cover simple usecases in as simplest way as possible
+    * I want to  be able to write complex structure as well
+    * I want to have it human readable
+    * I want to see some examples
+
+## Scope of project
+* filter test set for frameworks
+* create test coverage report
+
+### What we covers
+* __Test case filtering__, based on relevancy and tags
+* __Test coverage__ document and analysis.
+* __Example__  [component](examples/general-component/tests)
+* __Tooling__ to work with this example, to show abilities
+* __Modular__ various tools can define own items for each test and parse it how you want.
+
+## Out of scope
+* __NO schedule tests__ - it is in scope of framework: (eg. avocado, unittest, py.test, restraint) and also it is part of [Invoking test initiative](https://fedoraproject.org/wiki/Changes/InvokingTests)
+* __NO dependency solving__ - each framework has own dependency solving (eg. python pypi deps, rpm dependencies in specfile)
+    * optional scope: it can be part of scope here after some discussion, but just as an optional feature
+* __NO test linking__ - each framework has to known how to interpret filtered tests format (eg. set of: local files, python classes, URLs)
+    * optional scope: could be transform formats to format of selected framework (backend) (eg. download tests from URLs and store it locally)
+
+## How it works
+* Tree structure of metadata splitted to two types
+* can use one metadata file, or split metadata file to each test or use combination of both solution.
+* Two type of `metadata.yaml`
+    * __general__ - fully descriptive file for writing general info about component and testing and whatever you want - [metadata.yaml](examples/general-component/tests/metadata.yaml)
+    * __test__ - basic metadata for test, it has same value as any test in `tests` element [metadata.yaml](examples/general-component/tests/sanity/metadata.yaml)
+
+
+## Installation
+```
+sudo make install
+```
+
+## Self-Check
+```
+sudo make check
+```
+
+## Usage
+ * Two tools: `tmet-filter` and `tmet-agregator`
+ * Swithc to example directory `examples/general-component/tests` and try them
+
+## Config examples
+
+### Simple config
+As output there will be two tests independent on backend framework
+
+```
+document: metadata
+subtype: general
+import_tests:
+    - "/bin/true"
+    - "/bin/false"
+```
+
+### Simple Config with MTF Linters
+tag filetrs with imported test and enabled MTF modulelint and import all tests (relatively to base dir)
+
+```
+document: metadata
+subtype: general
+enable_lint: True
```

Comment:
I think this should be test, not an option

Reply:
not true generally. because this __lint__ is set of tests, on one test, thats why this is special case